components/azure-iot/devdoc/platform_openssl_compact_requirements.md

```diff
@@ -0,0 +1,89 @@
+platform_openssl_compact
```

Comment:
It's not entirely clear who is the intended audience of .md files under devdoc/? If those are the application developers, would you consider linking to this pages from ESP-IDF documentation under $IDF_PATH/docs? Otherwise these documents would be pretty hard to find.